lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -20,7 +20,13 @@ def initialize(engine_config:)
         end
 
         def run(file)
-          process_file(file)
+          $stderr.puts("About to analyze #{file}. If analysis fails, consider excluding this file.")
```

Comment:
I don't think we need to go so far as to say "If analysis fails", and this feels a little too wordy to me in general. Just saying `"Processing #{file}..."` is enough, IMO.